packages/at_commons/lib/src/keystore/at_key.dart

```diff
@@ -38,22 +38,31 @@ class AtKey {
   /// These keys will never be synced between the client and secondary server.
   bool _isLocal = false;
 
+  String? get namespace => _namespace;
+
+  set namespace(String? namespace) {
+    if (namespace != null && namespace.isNotEmpty) {
```

Comment:
Please replace the above IF conditions with "namespace.isNotNullOrEmpty" than having two checks.

Reply:
addressed as part of the last commit